tests/test_acquisition.py

```diff
@@ -6,7 +6,14 @@
 from wagl.constants import BandType
 from wagl.temperature import temperature_at_sensor
 
-from .data import LS5_SCENE1, LS7_SCENE1, LS8_SCENE1
+from .data import (
+    LS5_SCENE1,
+    LS7_SCENE1,
+    LS8_SCENE1,
+    LS8_SCENE1C2,
+    LS8_SCENERTC2,
+    LS7_SCENERTC2,
```

Comment:
Do these constants need a scene number in the name? e.g. `LS8_SCENE1RTC2`

Reply:
I dropped the 1.  I didn't think it was adding any value.

Reply:
Yeah, they could be dropped.  From memory, it was to support more than 1 acquisition from a given sensor, such as multiple versions.  But over time we've decided to stop supporting older versions of `scenes` that USGS would create as it was adding unnecessary complexity.  By older versions, I mean metadata, filenames, metadata keys are completely different, and we would attempt to support each version.  Not worth the hassle anymore.